src/pages/wirkung/[id].js

```diff
@@ -35,12 +35,15 @@ export default function WirkungPage() {
 			</div>
 
 			<button
+				disabled={
+					routine.filter(routine => routine.id === Number(id)).length > 0 ? true : false
+				}
```

Comment:
Please see my previous comment 🙂 

```suggestion
				disabled={
					routine.filter(routine => routine.id === Number(id)).length
				}
```

Reply:
Ok, erledigt.